libkbfs/folder_branch_ops.go

```diff
@@ -2231,6 +2248,12 @@ func (fbo *folderBranchOps) CreateFile(
 		entryType = File
 	}
 
+	if excl == WithEXCL {
+		if err = fbo.cr.Wait(ctx); err != nil {
```

Comment:
Good point! I think it's worth doing; if we support `O_EXCL`, it should be a guarantee.